envoy/config/filter/accesslog/v2/accesslog.proto

```diff
@@ -147,6 +147,20 @@ message ResponseFlags {
 
   // Indicates that the request was rate-limited locally.
   bool rate_limited = 12;
+
+  // Reasons why the request was unauthorized
+  enum UnauthorizedType {
+    // Default is permitted
+    PERMITTED = 0;
```

Comment:
Should this be under the `Unauthorized` message below? That way, there is no need to have a permitted default, it can just have for default `TYPE_NAME_UNSPECIFIED = 0` as per https://github.com/envoyproxy/data-plane-api/blob/master/STYLE.md?